roles/haproxy/files/haproxy.tmpl

```diff
@@ -1,12 +1,12 @@
 global
```

Comment:
This actually https://github.com/hashicorp/consul-template code with shouldn't has space